src/Avalonia.Controls/HotkeyManager.cs

```diff
@@ -37,14 +38,15 @@ class Manager
             private TopLevel _root;
             private IDisposable _parentSub;
             private IDisposable _hotkeySub;
+            private IDisposable _commandParameterChangedSubscriber;
             private KeyGesture _hotkey;
             private readonly HotkeyCommandWrapper _wrapper;
             private KeyBinding _binding;
 
             public Manager(IControl control)
             {
                 _control = control;
-                _wrapper = new HotkeyCommandWrapper(_control);
+                _wrapper = new HotkeyCommandWrapper(_control as ICommandSource);
```

Comment:
This can cause `null` to be passed to `HotkeyCommandWrapper`, which doesn't do null checks. Probably best to not create the wrapper if the control isn't an `ICommandSource`.

Reply:
Oh actually, I notice now that the check is at line 110. In that case, probably should use a standard cast instead of `as` to indicate that the type is already known to be correct.